src/components/OrgUpcomingEvents.tsx

```diff
@@ -0,0 +1,32 @@
+import { api } from "@src/utils/api";
+import { type FC } from "react";
+import { SelectClub as Club } from "@src/server/db/models";
+
+const cur_time = new Date();
+
+const OrgUpcomingEvents:FC<{club : Club}> = ({club}) => {
+  
+    const query_events = (api.event.byClubId.useQuery({ clubId: club.id, currentTime: cur_time, sortByDate: true  })).data
+    
+    return (
+      <div className="w-full rounded-lg bg-slate-100 p-10">
+        <h1 className="font-medium text-2xl mt-5">
+          Upcoming Events
+        </h1>
+        <div className="flex flex-col items-start justify-evenly  md:flex-row mt-5">
+            {query_events?.map( (event) => (
```

Comment:
Try adding some alternate text whenever there are no items in the array

Reply:
Here is the placeholder text if there is no item the array: 
![image](https://github.com/UTDNebula/jupiter/assets/90099089/4a5bb068-7c4b-4ee6-bcf8-645e2581d031)